docs/information/supported_adapters.md

```diff
@@ -31,11 +31,19 @@ Zigbee2MQTT officially supports the following adapters:
   </tr>
   <tr>
     <td><img src="../images/ZigStar-USB.jpg"></td>
-    <td>ZigStar CC2652P2 USB Coordinator</td>
-    <td>Open source hardware CC2652P based,USB connected adapter with external antenna</td>
-    <td><a href="https://github.com/Koenkk/Z-Stack-firmware/raw/master/coordinator/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_coordinator_20210120.zip">Coordinator</a><br/> <a href="https://github.com/Koenkk/Z-Stack-firmware/raw/master/router/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_router_20210128.zip">Router</a></td>
-    <td><a href="https://mercenaruss.github.io/zigbee-stick-v4/flashing">Flashing</a></td>
-    <td><a href="https://t.me/zigberu">Zigbee DYI Market (Russian)</a></td>
+    <td>ZigStar Stick v4</td>
+    <td>Powerfull Open source dongle with external antenna based on CC2652P</td>
+    <td><a href="https://github.com/Koenkk/Z-Stack-firmware/blob/master/coordinator/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_coordinator_20210708.zip">Coordinator</a><br/> <a href="https://github.com/Koenkk/Z-Stack-firmware/blob/master/router/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_router_20210128.zip">Router</a></td>
```

Comment:
Please use raw links for firmwares: https://github.com/Koenkk/Z-Stack-firmware/raw/master/coordinator/Z-Stack_3.x.0/bin/CC2652R_coordinator_20210708.zip

Reply:
Done